src/libopensc/card-epass2003.c

```diff
@@ -646,6 +658,7 @@ sha256_digest(struct sc_card *card, const unsigned char *input, size_t length, u
 
 	r = openssl_dig(md, input, length, output);
 	sc_evp_md_free(md);
+	sc_log_openssl(card->ctx);
```

Comment:
same here in most of the changes to epass -- this code path is executed in both positive and negative cases